contrib/hashicorp/consul/benchmark_test.go

```diff
@@ -0,0 +1,66 @@
+package consul
+
+import (
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	consul "github.com/hashicorp/consul/api"
+)
+
+func BenchmarkKV(b *testing.B) {
+	key := "test.key"
+	pair := &consul.KVPair{Key: key, Value: []byte("test_value")}
+	testCases := map[string](func(k *consul.KV) error){
```

Comment:
This is great! Thanks for changing them. Two minor things:
* Can you please post the difference here as a comment? It would be interesting to see the results of both of these benchmarks to get a good idea of the impact of tracing.
* Can you please remove the `testCases` variable and simply inline them? It's what we've been doing [elsewhere too](https://github.com/DataDog/dd-trace-go/blob/v1/contrib/internal/sqltest/sqltest.go#L52-L58).

Reply:
Here are the benchmark results (also in the PR body):


| Function | Untraced Execution Time (ms) | Traced Execution Time (ms) | Difference (ms) | Difference (%) |
|----------|------------------------------|----------------------------|-----------------|----------------|
| Put      | 1.371122                     | 1.437806                   | 0.066684        | 4.86%          |
| Get      | 1.042376                     | 1.121287                   | 0.078911        | 7.57%          |
| List     | 1.05023                      | 1.094199                   | 0.043969        | 4.19%          |
| Delete   | 1.347029                     | 1.453241                   | 0.106212        |  7.88%         |